src/signer/api/src/methods.rs

```diff
@@ -18,9 +20,11 @@ impl SignerMethods {
         // Note: Fees are determined with the aid of scripts/check-pricing
         match self {
             SignerMethods::GenericCallerEcdsaPublicKey
+            | SignerMethods::SchnorrPublicKey
             | SignerMethods::EthAddress
             | SignerMethods::EthAddressOfCaller => 1_000_000_000,
             SignerMethods::GenericSignWithEcdsa
+            | SignerMethods::SchnorrSign
             | SignerMethods::EthSignTransaction
             | SignerMethods::EthPersonalSign
             | SignerMethods::EthSignPrehash => 40_000_000_000,
```

Comment:
Note: There is a backlog task to check prices in CI.  Specifically it would count the instructions used, multiply it by the expected subnet size and a factor of something like 1.4 to cover the cost of attacks and errors such as requests made without payment, that do burn up a few cycles.